alembic/versions/1ed43776064f_initial_alter_statements.py

```diff
@@ -0,0 +1,44 @@
+"""Initial alter statements
+
+Revision ID: 1ed43776064f
+Revises: ff7e7ae6d711
+Create Date: 2017-04-23 19:06:40.644549
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '1ed43776064f'
+down_revision = 'ff7e7ae6d711'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+
+try:
+    is_sqlite = op.get_context().dialect.name == 'sqlite'
+except:
+    is_sqlite = False
+
+if is_sqlite:
+    op.get_context().connection.execute('PRAGMA foreign_keys=ON;')
+    utcnow_server_default = "(datetime('now', 'utc'))"
+else:
+    utcnow_server_default = "timezone('utc', current_timestamp)"
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
```

Comment:
In general, I'd prefer to not leave this kind of comment in here, since it looks like a ``TODO`` which was neglected, whereas in this case I believe that no further adjustments are needed.